render-conjure/src/main.rs

```diff
@@ -38,7 +38,9 @@ fn render(url: &str, prefix: &str, out_dir: &str) {
         .unwrap();
 
     // ensure the path is relative to the workspace root
-    let out_dir = Path::new(file!()).join("../../../..").join(out_dir);
```

Comment:
At some point the compiler started using relative paths for `file!()` so the old approach stopped working.